store/tikv/client.go

```diff
@@ -92,6 +98,12 @@ func (c *rpcClient) SendKVReq(ctx goctx.Context, addr string, req *kvrpcpb.Reque
 	start := time.Now()
 	defer func() { sendReqHistogram.WithLabelValues("kv").Observe(time.Since(start).Seconds()) }()
 
+	select {
```

Comment:
This code is duplicate with the code on the 60th line in `store/tikv/region_request.go`.